<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReservationMain</class>
 <widget class="QMainWindow" name="ReservationMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReservationMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>571</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="EN">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter a name for the reservation:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="rN">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="RD">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservation Date(s)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NEXT">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>560</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="SD">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="DD">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Duration (Days)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="dS">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>150</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
     </widget>
     <widget class="QLabel" name="OC">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>290</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Occupants (4 max)</string>
      </property>
     </widget>
     <widget class="QLabel" name="OA">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>310</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Adults:</string>
      </property>
     </widget>
     <widget class="QLabel" name="OCH">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Children: </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="oA">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="oC">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="rP">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>430</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Reserved Parking ($12.75 a night)</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="rS">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Choose Room Style</string>
      </property>
      <widget class="QRadioButton" name="S">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Standard</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="A">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Atrium</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="bA">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Choose Bed Arrangement </string>
      </property>
      <widget class="QRadioButton" name="sKB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Single King Bed</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="dQB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Double Queen Bed</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="oP">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>550</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Overall Price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="PR">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>550</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="selDate">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>321</width>
        <height>221</height>
       </rect>
      </property>
      <property name="selectedDate">
       <date>
        <year>2019</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2019</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="ocError">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="NEXT2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>560</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BACK">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>560</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>511</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Reservation Total</string>
      </property>
      <widget class="QLabel" name="rtR">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>47</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Room:</string>
       </property>
      </widget>
      <widget class="QLabel" name="rtT">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tax (15%): </string>
       </property>
      </widget>
      <widget class="QLabel" name="rtP">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Parking:</string>
       </property>
      </widget>
      <widget class="QLabel" name="rtRF">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Resort Fee ($15 per night):</string>
       </property>
      </widget>
      <widget class="QLabel" name="rtTl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Total: </string>
       </property>
      </widget>
      <widget class="QLabel" name="roomFee">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>30</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Room:</string>
       </property>
      </widget>
      <widget class="QLabel" name="tax">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Room:</string>
       </property>
      </widget>
      <widget class="QLabel" name="parkingFee">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Room:</string>
       </property>
      </widget>
      <widget class="QLabel" name="resortFee">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Room:</string>
       </property>
      </widget>
      <widget class="QLabel" name="total">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Room:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>511</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Payment Option</string>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>461</width>
         <height>81</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>VISA</string>
        </attribute>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Card Number:</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="VisaNum">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>MasterCard</string>
        </attribute>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Card Number:</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="MasterNum">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Discover</string>
        </attribute>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Card Number:</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="DiscovNum">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>American Express</string>
        </attribute>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Card Number:</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="AmerNum">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>261</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Expiration Date:</string>
       </property>
      </widget>
      <widget class="QLabel" name="cardError">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>110</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QSpinBox" name="expMonth">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>12</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="expYear">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2019</number>
       </property>
       <property name="maximum">
        <number>3019</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>160</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Month       Year</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservation Confirmed!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CLOSE">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>560</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Check in day:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nights reserved:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>150</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Room type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parking:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adults:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Children</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Payed with card ending in:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Thank you so much for doing buisness with us today!</string>
      </property>
     </widget>
     <widget class="QLabel" name="Ac">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adults:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Cid">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Check in day:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Cc">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Children</string>
      </property>
     </widget>
     <widget class="QLabel" name="Frt">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>150</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Room type:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Ip">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Parking:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Nnr">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nights reserved:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Clf">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>270</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Payed with card ending in:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Fttl">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Total:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
